.github/workflows/code.yml

```diff
@@ -11,9 +11,9 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v3
         with:
           node-version: "14.x"
```

Comment:
Should we update node-version in this file as well?

Reply:
I think we _may_ do that, but I don't think we _have to_. The requirement of using Node 16 applies to how particular actions are run underneath, not to what they are supposed to do. At least that's my understanding. So in our workflow we shouldn't use actions which are run using Node.js versions other than v16, but we can configure the workflow to execute a step that configures Node.js v14 on working environment and executes next steps on that version of the Node.js.

Do you want the node to be updated to 16 just so that we would be using latest version?